README.md

```diff
@@ -1,99 +1,143 @@
+...
+```
 
-Later on, there will be tests concerning `Tire` tread. Since that does not have to do with repeling liquid, it is not in the same scope.
+Expectations are the part of an example the verifies code has the expected behavior. Here was are saying we expect the value returned by sending the message `first_name` to the `person` object, to return an object that is equal to the string `"Expected First Name"`. There is a lot going on here, and the syntax used is hiding some of the details. Lets take a look at what this method is doing with the missing syntax add back in:
 
-## It Blocks
+```ruby
+expect(person.first_name).to(eq("Expected First Name"))
+```
 
-Rspec uses an `it` method to signify the place where actual tests go. `it` takes a description string and a block of test code.
+We could further expand this to:
 
 ```ruby
-Rspec.describe Tire do
-  describe "repels liquid" do
-    # ...
-
-    context "in rain storms" do
-      it "resists water" do
-        expect(Tire.new.through_rainstorm).to eq("Repelled water!")
-      end
-
-      it "resists mud" do
-        expect(Tire.new.through_muddy_rainstorm).to eq("Repelled mud!")
-      end
-
-      it "resists acid" do
-        expect(Tire.new.through_acid_rainstorm).to eq("Repelled acid!")
-      end
-    end
+# Capture the value returned by sending person the first_name message
+returned_first_name = person.first_name
 
-  # ...
+# Define a expectation object that we can make expectations of
+expectation_object = expect(returned_first_name)
+
+# Define a matcher for comparing the returned_first_name value
+matcher = eq("Expected First Name")
+
+
+# Put it all together, and make the expectation
+expectation_object.to(matcher)
 ```
 
+You can see we have a object being created with the `expect` method. This takes a value, and lets us then make expectations on the value it contains. Next we define a matcher. The matcher is sent to the expectation object via the `to` method, and the expectation object then check if the matcher holds true. If it does, the expectation passes, if it does not, the test will fail. You can read more about the built in [expectation and matching library of RSpec here](https://relishapp.com/rspec/rspec-expectations/docs).
+
+# File Naming Conventions
+
+It is best to end a spec file with `_spec.rb`, and to place it under the `spec` directory. This will make RSpec identify it, and make running it easier. There are some other conventions related to naming and location that are good to keep in mind. If the above `Person` was a file in a Ruby Gem type project and lived under `lib/gem_name/person.rb` folder, a good place to put this `Person` spec would be `spec/person_spec.rb`. If this was a Rails app, and the `Person` was a file that lived in `app/modles/person.rb`, a good place for this file would be `spec/modles/person_spec.rb`.
```

Comment:
> This will make RSpec identify it, and make running it easier

Would it be worth noting that all sorts of tooling assumes the convention that rspec specifications end with the `_spec.rb` suffix. - vim's spec runner package, emacs' rspec-mode, I think atom, sublime-text and others have the same assumption.

Reply:
How does this sound:

This will make RSpec, and tools we use to develop, identify it, and make running it easier